yield_plugin/include/yield_plugin/yield_plugin.hpp

```diff
@@ -60,6 +60,16 @@ struct PointSpeedPair
   double speed = 0;
 };
 
+/**
+ * \brief Convenience class for saving collision results
+ */
+struct CollisionData
+{
+  rclcpp::Time collision_time;
+  lanelet::BasicPoint2d predicted_vehicle_collision_point;
+  lanelet::BasicPoint2d predicted_obstacle_collision_point;
```

Comment:
The `get_collision()` or (`get_collision_data()`) function is agnostic to the trajectory types considering the `trajectory1` and `trajectory2` parameter names. Consider renaming to `point1` and `point2`, respectively. This may seem antithetical to clearly-named variables, but I think this is a good exception case. The `get_collision()` function is generic, so it's return type should be as well.

Reply:
Renamed